package-lock.json

```diff
@@ -8687,7 +8684,6 @@
       "version": "0.6.1",
       "resolved": "https://registry.npmjs.org/optimist/-/optimist-0.6.1.tgz",
       "integrity": "sha1-2j6nRob6IaGaERwybpDrFaAZZoY=",
-      "dev": true,
```

Comment:
I am quite confused as to why these `"dev": true`'s are being removed, but for now I'm just gonna trust `npm`.